backend/src/main/resources/application-dev.yaml

```diff
@@ -30,3 +30,4 @@ datahub:
   api-key: "not-a-valid-api-key"
   api-version: "2021-09-21"
   csv-upload-api-client: "simple_report"
```

Comment:
does this need to be updated to be `simple_report.csvuploader` as well?

Reply:
Even better, we can just remove it (and most of the stuff in this block). Everything but the URL can either be inherited from the defaults from `application.yaml` or is a secret and should be in the gitignored `application-local.yaml`. ([ffd6eef](https://github.com/CDCgov/prime-simplereport/pull/3766/commits/ffd6eef16767cc963125b8998aae13e2bd9fc04a))